cmd/osmosisd/cmd/root.go

```diff
@@ -147,12 +147,12 @@ var (
 		{
 			Section: "consensus",
 			Key:     "timeout_commit",
-			Value:   "500ms",
+			Value:   "550ms",
 		},
 		{
 			Section: "consensus",
 			Key:     "timeout_propose",
-			Value:   "1.8s",
+			Value:   "1.9s",
```

Comment:
Lets keep timeout propose where it is, otw LGTM!

Reply:
For context, this is is saying "if the proposer is offline, how long do we wait for round 2".

Our blocktime is 1.3s (maybe 1.5s), 1.8s is already way longer! (and actually comes later in the pipeline)

Reply:
updated to 1.6s, thanks for your input :bow: